globus_cli/commands/endpoint/show.py

```diff
@@ -22,6 +22,7 @@ def endpoint_show(endpoint_id):
     ("Display Name", "display_name"),
     ("ID", "id"),
     ("Owner", "owner_string"),
+    FormatField("Description", "description", wrap_enabled=True),
```

Comment:
Optional, but now that this is a fancy object rather than a tuple, it would be possible to write things like `FormatField("activated")` and have that mean what `("Activated", "activated")` means today.

I'd be amenable to such refinements if we like this direction. But the two-argument form will be needed at least for cases like `Owner, owner_string`.

Reply:
I like the direction, but I'd rather we keep the two-argument form even if the display name could be inferred from the field name to keep things simple and uniform.

Reply:
Agreed. I actually have a branch which tried doing `replace("_", " ").title()`, but it (1) doesn't save much code and (2) becomes confusing to read.